swiggy-pdp-cart/src/components/Footer/Socials/Socials.js

```diff
@@ -0,0 +1,18 @@
+import facebook from "../../../images/facebook.png";
+import instagram from "../../../images/instagram.png";
+import pinterest from "../../../images/pinterest.png";
+import twitter from "../../../images/twitter.png";
+import SocialImage from "./SocialsImage/SocialsImage";
+
+export default function Socials() {
+    return (
+      <div className="col-4">
+        <ul>
+          <SocialImage src={facebook} alt="facebook logo" />
```

Comment:
Whenever performing repetitive stuff, always follow DRY.

Convert the social links information into an array of objects. 

Example: `[{ id: 0, src: '', alt: ''}]` and render the social links based on this JSON.

Addition/deletion of a link then should just be a simple config update instead of updating the code again and again